napari/layers/utils/interaction_box.py

```diff
@@ -77,7 +77,7 @@ def generate_transform_box_from_layer(
     np.ndarray
         Vertices and handles of the interaction box in data coordinates.
     """
-    bounds = layer._display_bounding_box_augmented(dims_displayed)
+    bounds = layer._display_bounding_box_augmented(np.asarray(dims_displayed))
```

Comment:
Why not `list(...)` here? That seems to be what `Layer. _display_bounding_box_augmented` wants?